packages/wdio-crossbrowsertesting-service/src/service.js

```diff
@@ -146,23 +146,17 @@ export default class CrossBrowserTestingService {
         return this.updateJob(oldSessionId, this.failures, true, browserName)
     }
 
-    updateJob (sessionId, failures, calledOnReload = false, browserName) {
-        return new Promise((resolve, reject) => request.put(this.getRestUrl(sessionId), {
-            json: true,
-            auth: {
-                user: this.cbtUsername,
-                pass: this.cbtAuthkey
-            },
-            body: this.getBody(failures, calledOnReload, browserName)
-        }, (e, res, body) => {
-            /* istanbul ignore if */
-            this.failures = 0
-            if (e) {
-                return reject(e)
-            }
-            global.browser.jobData = body
-            return resolve(body)
-        }))
+    async updateJob (sessionId, failures, calledOnReload = false, browserName) {
+        const json = this.getBody(failures, calledOnReload, browserName)
+        this.failures = 0
+        const response = await got.put(this.getRestUrl(sessionId), {
+            json,
+            responseType: 'json',
+            auth: `${this.cbtUsername}:this.cbtAuthkey`
```

Comment:
I think it should be `${this.cbtAuthkey}`

Reply:
good catch 👍